salah/app/src/main/java/com/tazzix/wear/salah/format.kt

```diff
@@ -46,6 +52,7 @@ fun Context.getAddressDescription(location: MyLocation, resolveLocality: Boolean
     if (locationVal.isEmpty()) locationVal = String.format("%.3f, %.3f", location.latitude, location.longitude)
 
     val today = SimpleDate(GregorianCalendar())
+    // Subtract 1 from DST due to bug in library: https://github.com/ahmedeltaher/Prayer-Times-Android-Azan/issues/27#issue-866377567
```

Comment:
Whoops, forgot to delete this line. I'll likely be pushing some more changes and can remove in another PR.